keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components/README.md

```diff
@@ -290,6 +294,12 @@ all existing klog features.
 - 20/06/2021 - Original proposal created in https://github.com/kubernetes/kubernetes/issues/99270
 - 30/07/2021 - KEP draft was created
 - 26/08/2021 - Merged in provisional state
+- 09/09/2021 - Merged as implementable
+- Kubernetes 1.23 (tenatative): alpha, [deprecation
+  period](https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-flag-or-cli)
+  starts
+- Kubernetes 1.24 (tentative): beta
+- Kubernetes 1.25 (tentative): GA, deprecated flags get removed
```

Comment:
I hope I understood the "available for two releases" rule correctly (https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-flag-or-cli): 1.23 and 1.24 still have them, 1.25 won't.

Reply:
We'll also have to check the "for one year" rule. But we can discuss that when we get closer to the actual removal.

Reply:
I'd prefer to remove them as soon as possible because the code for supporting them during the transition is really ugly. Can we keep 1.25 as tentative and check again when we are closer to that to determine whether we can go ahead?

Reply:
Per the linked policy + discussion with @logicalhan on today's triage, https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-flag-or-cli states 12 months or 2 releases, whichever is _longer_ - I think it has to be 1.26.

Reply:
I find that rule ambiguous because it's unclear which releases are counted for that "2 releases" check.

But let's be conservative and aim for 1.26. Commit and PR description was updated.